lib/commands/gesture.js

```diff
@@ -79,6 +82,13 @@ function isDoubleTap (gestures) {
   return false;
 }
 
+function isTwoFingerTap (gestures) {
+  if (gestures.length === 1 && gestures[0].action.toLowerCase() === 'twofingertap') {
```

Comment:
Technically nothing of this is in the W3C spec, so we could add it. Otherwise it could be mapped from a two tap gestures in the multi-action api.